tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+      "baseUrl": ".",
+      "paths": {
+          "@src/*": ["src/*"]
+      },
+      "target": "es6",
+      "module": "commonjs",
+      "moduleResolution": "node",
+      "rootDir": "./src",
+      "outDir": "dist/scripts",
+      "strict": true,
+      "sourceMap": true,
+      "jsx": "react",
```

Comment:
We should not hint that react is something we want to support in this extension
```suggestion
```

Reply:
Same as above. Will fix.